platform-tests/example-apps/healthcheck/db.go

```diff
@@ -24,18 +26,44 @@ func dbHandler(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
-func testDBConnection(ssl bool) error {
-	dbURL, err := url.Parse(os.Getenv("DATABASE_URL"))
+func testDBConnection(ssl bool, service string) error {
+	var sslTrigger = "sslmode"
+	var sslOn = "verify-full"
+	var sslOff = "disable"
+
+	dbu := os.Getenv("DATABASE_URL")
+
+	switch service {
```

Comment:
I find this hard to read because it combines:
- setting the default service to be postgres
- setting the various db specific parameter names for MySQL, whereas the postgres ones are set as defaults

How about:
- first setting postgres as the default service (above this switch statement
- moving the postgres settings into the switch statement
?